flight/PiOS/Common/pios_bmp280.c

```diff
@@ -200,6 +195,36 @@ int32_t PIOS_BMP280_Init(const struct pios_bmp280_cfg *cfg, int32_t i2c_device)
 	return 0;
 }
 
+/**
+ * Initialise the BMP280 sensor
+ */
+#ifndef PIOS_EXCLUDE_BMP280_I2C
+int32_t PIOS_BMP280_Init(const struct pios_bmp280_cfg *cfg, int32_t i2c_device)
+{
+	dev = (struct bmp280_dev *) PIOS_BMP280_alloc();
+	if (dev == NULL)
+		return -1;
+
+	dev->i2c_id = i2c_device;
+
+	return PIOS_BMP280_Init_Common(cfg);
+}
+#endif /* PIOS_EXCLUDE_BMP280_I2C */
+
+#ifdef PIOS_INCLUDE_BMP280_SPI
+int32_t PIOS_BMP280_Init_SPI(const struct pios_bmp280_cfg *cfg, int32_t spi_device,
```

Comment:
I did `PIOS_*_SPI_Init` in pios_mpu. Seems better to me to keep the least general part of the symbol at the end. WDYT? Also applies to the other I2C/SPI specific functions.